_ng/server/sub_generators_endpoint.js

```diff
@@ -38,4 +38,26 @@ exports.EndpointSubGenerator = class EndpointSubGenerator {
         case ServerFactory.tokens().NODE: NodeFactory.build(this.wrapper).copyFiles(); break;
     }
   }
-}
+
+  end() {
+    const routesFile = 'server/routes/index.js';
```

Comment:
That'll only work for Node. 

Maybe check `this.wrapper.server === ServerFactory.tokens().NODE`?

And for Go, that'd be `server/routes/routes.go`.